conan/cli/commands/create.py

```diff
@@ -46,10 +47,13 @@ def create(conan_api, parser, *args):
 
     out = ConanOutput()
     out.highlight("Exporting the recipe")
-    ref = conan_api.export.export(path=path,
+    ref, conanfile = conan_api.export.export(path=path,
                                   name=args.name, version=args.version,
                                   user=args.user, channel=args.channel,
                                   lockfile=lockfile)
+
+    is_build_require = args.build_require
+    is_build_require = is_build_require or getattr(conanfile, "is_build_require", False)
```

Comment:
is/should this `conanfile. is_build_require` somehow related to the `node. is_build_require`?

Reply:
``node.is_build_require`` doesn't exist?